projects/common.py

```diff
@@ -27,11 +27,17 @@ def install_packages(packages, proxy=''):
         sudo('yum -y install {0}'.format(' '.join(packages)), pty=False)
 
 
+def update_packages(packages, proxy=''):
+    with shell_env(http_proxy=proxy, ftp_proxy=proxy):
+        sudo('yum -y update {0}'.format(' '.join(packages)), pty=False)
```

Comment:
I added the yum command as you offered, thanks!

regarding the proxy support - you said you'll move it to the env.